src/lib/AutoComplete.js

```diff
@@ -0,0 +1,243 @@
+/*
```

Comment:
Note to other reviewers: This lib was not written by us, but taken from the below source. It was only adapted to satisfy our typescript and eslint rules.